virtcontainers/utils/utils.go

```diff
@@ -203,3 +206,16 @@ func BuildSocketPath(elements ...string) (string, error) {
 
 	return result, nil
 }
+
+// SupportsVsocks returns true if vsocks are supported, otherwise false
+func SupportsVsocks() bool {
+	if _, err := os.Stat(VSockDevicePath); err != nil {
```

Comment:
As a debug info, we could print the error.

Reply:
we don't have a logger in `utils.go`